kura/distrib/src/main/resources/fedora25/README.md

```diff
@@ -0,0 +1,26 @@
+Manual steps to install Kura 3.0 on Fedora 25 / Raspberry Pi
+==
+
+1. Download Fedora Minimal image from [here](https://arm.fedoraproject.org/).
+1. Copy downloaded image to your sdcard. There are many references on how to do this for your development platform,
+such as the [Raspberry Pi installation guide](https://www.raspberrypi.org/documentation/installation/installing-images/)
+      The command for the Fedora 25 image on Linux or Mac is something like:
+      ```
+      xzcat ~/Downloads/fedora/Fedora-Minimal-armhfp-25-1.3-sda.raw.xz | sudo dd bs=1m of=/dev/rdisk2
+      ```
+1. Load the sdcard above and boot the RPi.
+1. Scan your network for the RPi (if using DHCP).
+      ```
+      sudo nmap -sP 192.168.0.0/24 | awk '/^Nmap/{ip=$NF}/B8:27:EB/{print ip}'
+      ```
+1. SSH into the RPi. You will need to be plugged into a wired ethernet network at this point,
+but after this step you could configure wifi and repeat the discovery step.
+1. Grow the root file system as described [here](https://fedoraproject.org/wiki/Architectures/ARM/F25/Installation#Resize_the_Root_Filesystem)
```

Comment:
I would add a reference to the section named "How do I enlarge the root partition?" in the FAQ. It was useful for me as I was using the command-line and I had no clue on how to expand the root partition using only commands.